### PR TITLE
Remove privileged=true and uid=0 on most services.

### PR DESCRIPTION
**Pull request by Quentin-M**
nova-compute, nova-libvirt, neutron-agents and Open vSwitch are left
untouched for now. A follow-up commit should try to limit their caps.

### CODE REVIEW

**Comment by DTadrzak** [comment]
> Entrypoint INFO: 2016/10/13 09:14:06 config.go:67: /etc/rabbitmq/rabbitmq-clusterer.config
Entrypoint WARNING: 2016/10/13 09:14:06 entrypoint.go:52: Resolving dependency for /etc/rabbitmq/rabbitmq-clusterer.config failed: Couldn't touch file /etc/rabbitmq/rabbitmq-clusterer.config: open /etc/rabbitmq/rabbitmq-clusterer.config: permission denied

> Entrypoint INFO: 2016/10/13 09:14:08 config.go:67: /etc/rabbitmq/rabbitmq.config
Entrypoint WARNING: 2016/10/13 09:14:08 entrypoint.go:52: Resolving dependency for /etc/rabbitmq/rabbitmq.config failed: Couldn't touch file /etc/rabbitmq/rabbitmq.config: open /etc/rabbitmq/rabbitmq.config: permission denied

> Entrypoint INFO: 2016/10/13 09:14:08 config.go:67: /etc/rabbitmq/rabbitmq-clusterer.config
Entrypoint WARNING: 2016/10/13 09:14:08 entrypoint.go:52: Resolving dependency for /etc/rabbitmq/rabbitmq-clusterer.config failed: Couldn't touch file /etc/rabbitmq/rabbitmq-clusterer.config: open /etc/rabbitmq/rabbitmq-clusterer.config: permission denied

> Entrypoint INFO: 2016/10/13 09:14:08 config.go:67: /etc/rabbitmq/rabbitmq-env.conf
Entrypoint WARNING: 2016/10/13 09:14:08 entrypoint.go:52: Resolving dependency for /etc/rabbitmq/rabbitmq-env.conf failed: Couldn't touch file /etc/rabbitmq/rabbitmq-env.conf: open /etc/rabbitmq/rabbitmq-env.conf: permission denied

**Comment by DTadrzak** [comment]
@Quentin-M Can you remove rabbitmq from this PR?
I think that Tomasz's PR will fix it (#104)

**Comment by Quentin-M** [comment]
Interesting, I didn't notice that. Good catch. I'll remove RabbitMQ from there!

**Comment by Quentin-M** [comment]
Updated!

File: glance/templates/api/deployment.yaml.j2 [hunk 1]
```diff
@@ -7,9 +7,6 @@ spec:
       labels:
         app: glance-api
     spec:
-      # TODO(DTadrzak): it must be removed in future
```
Comment (ss7pro) [inline glance/templates/api/deployment.yaml.j2]: Hey,

/etc/ceph/ceph.conf directory is not writeable, by root. We need to fix that at the image level @DTadrzak

Comment (DTadrzak) [inline glance/templates/api/deployment.yaml.j2]: New images pushed on quay.io. fyi tag newton.

**Comment by DTadrzak** [comment]
LGTM

**Comment by ss7pro** [comment]
LGTM

**Pull request merged**